sql/core/src/test/scala/org/apache/spark/sql/TPCDSBase.scala

```diff
@@ -21,6 +21,47 @@ import org.apache.spark.sql.catalyst.TableIdentifier
 import org.apache.spark.sql.internal.SQLConf
 import org.apache.spark.sql.test.SharedSparkSession
 
+
+/**
+ * Base trait for TPC-DS related tests.
+ *
+ * Datatype mapping for TPC-DS and Spark SQL, see more at:
+ *   http://www.tpc.org/tpc_documents_current_versions/pdf/tpc-ds_v2.9.0.pdf
+ *
+ *    |---------------|---------------|
+ *    |    TPC-DS     |  Spark  SQL   |
+ *    |---------------|---------------|
+ *    |  Identifier   |      INT      |
+ *    |---------------|---------------|
+ *    |    Integer    |      INT      |
+ *    |---------------|---------------|
+ *    | Decimal(d, f) | Decimal(d, f) |
+ *    |---------------|---------------|
+ *    |    Char(N)    |    Char(N)    |
+ *    |---------------|---------------|
+ *    |  Varchar(N)   |  Varchar(N)   |
+ *    |---------------|---------------|
+ *    |     Date      |     Date      |
+ *    |---------------|---------------|
+ *
+ * Remarks:
```

Comment:
these remarks don't seem to be useful. It's more important to mention that the table schema matches the official tpcds toolkit.